android-components/components/lib/crash/build.gradle

```diff
@@ -60,6 +62,7 @@ android {
 dependencies {
     implementation Dependencies.kotlin_stdlib
     implementation Dependencies.kotlin_coroutines
+    implementation Dependencies.kotlin_json
```

Comment:
I did consider the builtin android Json utilities, and also looked (in vain) for other "batteries included" options (aside from `kotlinx`). Like you pointed out, it would be extra work to write serialization boilerplate, and I wanted to leave it as easy as possible to add new data in the future.